app/helpers/layout_helper.rb

```diff
@@ -4,9 +4,15 @@ def active_org(org)
   end
 
   def colorize_transaction(transaction)
-    css_class = colorize_amount(transaction.amount)
-    if params[:controller] == 'home' && current_member
-      if current_member.last_visited_at
+    if transaction.category && transaction.category.name == 'Receipt'
```

Comment:
@yunixon just add to transaction model method `transfer?` and use it everywhere. In this method compare category by id. SO it will be something like this:
```rb
def transfer?
  category_id == Category.receipt_id
end
```
in Category
```
def self.receipt_id
  @receipt_id ||= find_by(name: CATEGORY_TRANSFER_...)
end
```